chickadee/io.py

```diff
@@ -49,10 +49,201 @@
     data_type="positiveInteger",
 )
 
-end_date = LiteralInput(
-    "end_date",
-    "End Date",
-    abstract="Defines the end of the calibration period",
-    default=date(2005, 12, 31),
-    data_type="date",
-)
+
+general_options_input = [
+    LiteralInput(
+        "units_bool",
+        "Units Boolean",
+        abstract="Whether to check the input units and convertthem to the target output units",
+        default=True,
+        data_type="boolean",
+    ),
```

Comment:
A minor suggestion; I think it would be nice to limit the maximum number of input occurrences to 1 for all of the newly added inputs.

Reply:
Good idea